app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -223,110 +212,74 @@ object TelemetryWrapper {
 
     @JvmStatic
     fun init(context: Context) {
-        // When initializing the telemetry library it will make sure that all directories exist and
-        // are readable/writable.
-        val threadPolicy = StrictMode.allowThreadDiskWrites()
-        try {
-            val resources = context.resources
 
-            val telemetryEnabled = isTelemetryEnabled(context)
-
-            val configuration = TelemetryConfiguration(context)
```

Comment:
Are you sure you want to get rid of legacy telemetry in this PR? We recommend having a bit of overlap to at least perform some validation (unless your team made a conscious decision about this). See our [migration guidelines](https://docs.google.com/document/d/1k7GabrWCDKp7Ae8mqr-XFnFxIuNlxsJM52D-1uVmgb0/edit#heading=h.5m3528mtfs3e).

Reply:
Thanks for this. I hadn't seen this doc before and it was pretty helpful. :)